app/views/layouts/_footer.html.erb

```diff
@@ -1,41 +1,37 @@
 <footer class="crayons-footer print-hidden">
-  <div id="footer-container" class="crayons-layout crayons-footer__container">
-    <div class="crayons-footer__info">
-      <div><%= render "layouts/logo" %></div>
-      <p><%= SiteConfig.community_description %></p>
-      <div><%= render partial: "layouts/social_media" %></div>
-      <%# The following copy is necessary for compatability with the Forem AGPL licence which requires instances to link back to the source. %>
-      <p class="fs-xs color-base-60"><%= community_qualified_name %> copyright <%= copyright_notice %></p>
-      <p class="fs-xs color-base-60">Built on <a href="https://www.forem.com">Forem</a> — the <a href="https://dev.to/t/opensource">open source</a> software that powers <a href="https://dev.to">DEV</a> and other inclusive communities.</p>
-      <p class="fs-xs color-base-60">Made with love and <a href="https://dev.to/t/rails">Ruby on Rails</a>.</p>
-    </div>
-
-    <nav class="crayons-footer__nav">
-        <a href="/" class="crayons-link crayons-link--block">Home</a>
-        <% NavigationLink.ordered.each do |link| %>
-          <% if !link.display_only_when_signed_in || (link.display_only_when_signed_in && user_signed_in?) %>
-            <a href="<%= link.url %>" class="crayons-link crayons-link--block">
-              <%= link.name %>
-            </a>
-          <% end %>
+  <div id="footer-container" class="crayons-layout crayons-layout--limited-m crayons-footer__container">
+    <nav class="m:-mt-4">
+      <a href="/" class="crayons-link inline-block p-2">Home</a>
+      <% NavigationLink.ordered.each do |link| %>
+        <% if !link.display_only_when_signed_in || (link.display_only_when_signed_in && user_signed_in?) %>
```

Comment:
Boolean columns automatically create predicate methods, so you can also use `link.display_only_when_signed_in?`.

Reply:
Would you mind suggesting an actual change here? It's not really my piece of code above and I'm a bit afraid I will screw something up if I misunderstood anything.

Reply:
Sure, should have done that from the beginning:

```suggestion
        <% if !link.display_only_when_signed_in? || (link.display_only_when_signed_in? && user_signed_in?) %>
```

Reply:
Out of curiosity - why using _this_ instead of the original version? It's longer (2 characters, I know lol :P) and seems to be doing the same thing, right?